packages/expo-image/android/build.gradle

```diff
@@ -34,6 +34,12 @@ buildscript {
 android {
   compileSdkVersion safeExtGet('compileSdkVersion', DEFAULT_COMPILE_SDK_VERSION)
   buildToolsVersion safeExtGet('buildToolsVersion', DEFAULT_BUILD_TOOLS_VERSION)
+
+  compileOptions {
```

Comment:
Could make sense to move these Java changes to another PR

Reply:
makes sense. moving these changes to #12939